src/exemplars.js

```diff
@@ -13,13 +13,16 @@ class Exemplar {
   constructor(type) {
     this.type = type;
     this.images = [];
+    this.isRand = type.startsWith("rand_");
+    this.type = type.replace("rand_", "");
```

Comment:
Are we assigning type twice now? I think we might be able to get rid of line 14

Reply:
Whoops yeah i forgot to remove that